rust/core-lib/src/session.rs

```diff
@@ -0,0 +1,54 @@
+use xi_rope::rope::RopeInfo;
+use xi_rope::delta::Delta;
```

Comment:
you can just import `xi_rope::RopeDelta`, which is an alias for `Delta<RopeInfo>`.